### PR TITLE
Check watched movie

### PR DESCRIPTION
**Pull request by Sven-Lyco**
- #10

### CODE REVIEW

File: src/components/ButtonCheckMovie.spec.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ButtonCheckMovie from './ButtonCheckMovie';
+
+describe('ButtonCheckMovie', () => {
+  it('renders', () => {
```
Comment (FlyingMoe91) [inline src/components/ButtonCheckMovie.spec.js:6]: "it renders" 😄 
what does it render?  :D

File: src/hooks/useIsAdult.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { useEffect, useState } from 'react';
+import { saveToLocal, loadFromLocal } from '../utils/localStorage';
+
+export default function useIsAdult() {
+  const navigate = useNavigate();
+  const [isAdult, setIsAdult] = useState(loadFromLocal('isAdult') ?? false);
+  useEffect(() => {
+    saveToLocal('isAdult', isAdult);
+    isAdult ? navigate('./serien') : navigate('./');
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (FlyingMoe91) [inline src/hooks/useIsAdult.js:11]: Not sure if you need it. Just wanted to mention it in case you missed it.

**Review by FlyingMoe91 — COMMENTED** [review]
Great work 👍

File: src/components/ButtonCheckMovie.spec.js (hunk 1 above)
Comment (Sven-Lyco) [reply src/components/ButtonCheckMovie.spec.js:6]: I have written it more precisely now :)

File: src/hooks/useIsAdult.js (hunk 2 above)
Comment (Sven-Lyco) [reply src/hooks/useIsAdult.js:11]: Yes, i need it. It's a setting for ES-Lint :)

**Pull request merged**